users/models.py

```diff
@@ -1,9 +1,21 @@
+from django.contrib.auth.models import UserManager
 from django.db import models
 from django.utils.translation import gettext_lazy as _
 from helusers.models import AbstractUser
 
 
+class CustomUserManager(UserManager):
+    def get(self, *args, **kwargs):
```

Comment:
The intent here was to try to optimise the permission check a bit so that it would not need to fetch the user orgs every time.